README.md

```diff
@@ -318,6 +319,56 @@ The Realm Dart package is `realm_dart`
 
 * For more usage of Realm Dart see the Realm Flutter usage above.
 
+# Device Sync with Flutter/Dart SDK
+
+Run the Realm with [Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and connecting to [Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/).
+
+# Atlas App Services Configuration Steps
+
+* Create an account on [realm.mongodb.com](https://realm.mongodb.com) - follow the instructions: [Get Started with Atlas](https://www.mongodb.com/docs/atlas/getting-started) or [Get Started with a template app](https://www.mongodb.com/docs/atlas/app-services/tutorial/flutter/)
+* Create a new app following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui) or [Create an app with a template app](https://www.mongodb.com/docs/atlas/app-services/tutorial/flutter/).
+* Read [Authentication Providers](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) to see how to configure the appropriate authentication provider type.
+* Go to the `Device Sync` menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+* [Find and Copy the App ID](https://www.mongodb.com/docs/atlas/app-services/reference/find-your-project-or-app-id/) of your new application.
```

Comment:
```suggestion
1. Read [Authentication Providers](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) to see how to configure the appropriate authentication provider type.
1. Go to the **Device Sync** menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
1. [Find and Copy the App ID](https://www.mongodb.com/docs/atlas/app-services/reference/find-your-project-or-app-id/) of your new application.
```

Reply:
by doing this, it renders as a numbered list. you could also do `1. `, `2. `, etc.

Reply:
https://stackoverflow.com/questions/64918160/markdown-how-to-make-numbered-list-without-writing-the-actual-numbers

Reply:
nice tip. Still if we go with a list here it would be better imo if we use the 1,2,3 since its more readable while editing the file as well.